include/Ark/VM/State.hpp

```diff
@@ -82,7 +83,7 @@ namespace Ark
          * @param name the name of the function in ArkScript
          * @param function the code of the function
          */
-        void loadFunction(const std::string& name, Value::ProcType function) noexcept;
+        void loadFunction(const std::string& name, Procedure::CallbackType function) noexcept;
```

Comment:
We should pass `function` as an rvalue and use `std::move` instead of copying it here